kythe/cxx/common/kzip_writer_test.cc

```diff
@@ -185,7 +193,7 @@ TEST(KzipWriterTest, IncludesDirectoryEntries) {
       // We don't really care about the rest of the entries, but it's easy
       // enough to fix the order of the subdirectories and minimally harmful.
       ElementsAre(
-          "root/", "root/units/", "root/files/",
+          "root/", "root/files/", "root/units/",
```

Comment:
Why did this order change?

Reply:
Because of the changes in `InitializeArchive`. It used to be 

`for (const auto name : {kRoot, kUnitRoot, kFileRoot})`

but now it's dynamically constructed, but `kRoot` and `kFileRoot` are always there (and at the front).